### DISCUSSION TITLE
Upgrade v2.3.3 to v.2.3.4

### DISCUSSION THREAD

**Discussion by jmp2000** [aqualinkd/AqualinkD#289] [Q&A] [answered]
Good morning! 

Getting started with AqualinkD (RPi Zero 2 W) on Home Assistant.    Have v2.3.3 up, and trying to follow the Wiki directions to upgrade to v2.3.4 which says:

cd ~/software/AqualinkD
git pull
sudo ./release/install.sh

However, when I'm logged in this is what happens, and it doesn't upgrade.  Any suggestions?

root@aqualinkd:~/software/AqualinkD# git pull
bash: git: command not found
root@aqualinkd:~/software/AqualinkD# sudo ./release/install.sh
AqualinkD config exists, did not copy new config, you may need to edit existing! /etc/aqualinkd.conf
AqualinkD defaults exists, did not copy new defaults to /etc/default/aqualinkd
Avahi/mDNS defaults exists, did not copy new defaults to /etc/avahi/services//aqualinkd.service
AqualinkD web config exists, did not copy new config, you may need to edit existing /var/www/aqualinkd//config.js
Starting daemon aqualinkd
root@aqualinkd:~/software/AqualinkD#

**Comment by sfeakes** [comment, marked as answer]
That looks like it upgraded, the messages are just letting you know it didn’t override your configuration.
